tests/unit/mixins/dirty-confirm-route-test.js

```diff
@@ -142,3 +142,26 @@ test('model is array and checkModelForDirty returns ROLLBACK, checkModelForDirty
   assert.ok(args[3]);
   assert.ok(result);
 });
+
+test('Rollback Dirty Model', function(assert) {
+  var name = 'George',
+    fullName = 'George Castanza';
+
+  var model = Ember.Object.create({
+    name: name,
+    isDirty: false,
+    isSaving: false,
+    rollback: function() {
+      this.name = name;
```

Comment:
This can just be `assert.ok(true);`, because you don't want to implement rollback, just see that it was called. Then put an `assert.expect(1);` to ensure it was called.